sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/NamedVIPSpec.java

```diff
@@ -75,6 +75,23 @@ public static Builder newBuilder() {
     return new Builder();
   }
 
+  public static Builder newBuilder(NamedVIPSpec copy) {
```

Comment:
I think we can get rid of this if this is not used anywhere.

Reply:
I wanted to leave them in for completeness sake :man_shrugging: